templates/zerver/help/sidebar.md

```diff
@@ -110,6 +110,7 @@
 * [Change your organization's name](/help/change-your-organizations-name)
 * [Change your organization's description](/help/change-your-organizations-description)
 * [Change your organization's avatar](/help/change-your-organizations-avatar)
+* [Delete a message](/help/delete-a-message)
```

Comment:
I would put this in the "Editing messages" section.  It's only sorta temporarily an admin-only feature.